### PR TITLE
Fix missing metadata in messages

### PR DESCRIPTION
**Pull request by adybbroe**
We were missing `variant` in output messages. This is now fixed.
Also all metadata (except `description`) in the hook is now being added as a dict, like this example (`pps_hooks.yaml`):
```
pps_hook:
    post_hook: !!python/object:nwcsafpps_runner.pps_posttroll_hook.PPSMessage
      description: "This is a pps post hook for PostTroll messaging"
      metadata:
        posttroll_topic: "PPSv2018"
        station: "norrkoping"
        output_format: "CF"
        level: "2"
        variant: DR
```
This means it is easier to add new metadata to the message if needed.

Also fixing a couple of bugs from previous PR #13

### CODE REVIEW

File: nwcsafpps_runner/pps_posttroll_hook.py [hunk 1]
```diff
@@ -185,10 +192,22 @@ def create_message(self, status, mda):
                 LOG.debug("Filename = %s", filename)
                 to_send['uid'] = os.path.basename(filename)
 
-        to_send['data_processing_level'] = self.level
-        to_send['format'] = self.output_format
-        to_send['status'] = status
-        station = self.station
+        # level, output_format and station are all required fields
+        for attr in ['level', 'output_format', 'station']:
+            if not attr in self.metadata:
```
Comment (TAlonglong) [inline nwcsafpps_runner/pps_posttroll_hook.py:197]: I would have writte ` if attr not in self.metadata:` but I guess both do the same

**Review by TAlonglong — APPROVED** [review]
LGTM

File: nwcsafpps_runner/pps_posttroll_hook.py (hunk 1 above)
Comment (adybbroe) [reply nwcsafpps_runner/pps_posttroll_hook.py:197]: Ah, yeah, right @TAlonglong , looks a bit awkward now that you say it!

Comment (adybbroe) [reply nwcsafpps_runner/pps_posttroll_hook.py:197]: I was too late seeing this - merged already. Sorry! I will fix next time if I remember!

**Pull request merged**